samples/bluetooth/nrf_auraconfig/src/nrf_auraconfig.c

```diff
@@ -212,27 +214,40 @@ static void subgroup_send(struct stream_index stream_idx)
 {
 	int ret;
 	static int prev_ret;
+	struct audio_data audio_frame = {0};
 
```

Comment:
Why clear the structure?
No assumptions about the values in the structure are made. All elements used in the module processing the data need to be filled correctly, with those unuse as don't care. This also in the majority of cases cause a failure for data unset. The user will then have to decide if it is really used or the data has ended up in the wrong place.

Reply:
It does matter for the locations, since we do an or-operation. Also I find it cleaner to not have random values set in case a module later on tries to use a value that is not set here.